src/ngraph/pass/constant_folding.cpp

```diff
@@ -1156,3 +1158,108 @@ void pass::ConstantFolding::construct_constant_product()
         make_shared<pattern::Matcher>(convert_op, "ConstantFolding.ConstantProduct");
     this->add_matcher(convert_matcher, constant_product_callback, all_pass_property_off);
 }
+
+template <typename T>
+static shared_ptr<op::Constant> fold_constant_concat_helper(const shared_ptr<op::Concat>& concat)
+{
+    auto concat_args = concat->get_arguments();
```

Comment:
Could you switch this to `inputs()` so we have less `get_arguments()` to deal with?